scripts/migrate.js

```diff
@@ -0,0 +1,60 @@
+import AWS from 'aws-sdk';
+import Redis from 'ioredis';
+import { info } from 'heroku-logger';
+
+import Link from '../src/Models/Link';
+import { Client as PostgreSQL } from 'pg';
+import { filesystem } from '../src/helpers';
+
+// Configure storage connections for AWS, Redis, and RDS:
+AWS.config.update({ region: 'us-east-1' });
+
+const redis = new Redis(process.env.REDIS_URL);
+
+const sql = new PostgreSQL({
+  host: process.env.DB_HOST,
+  user: process.env.DB_USER,
+  password: process.env.DB_PASSWORD,
+  database: 'bertly_clicks',
+  port: 5432,
+});
+
+(async () => {
+  await sql.connect();
+
+  // Iterate over our existing links and migrate them to Bertly 2.0's storage:
+  const stream = redis.scanStream({ match: 'bertly:key:*', count: 100 });
+  for await (const chunk of stream) {
+    for (const link of chunk) {
+      const key = link.replace('bertly:key:', '');
+      const url = await redis.get(link);
```

Comment:
One thing I realized I haven't thought about yet is [URL normalization](https://github.com/DoSomething/bertly/blob/7dc66f7a1127ad6721653edd6ee421dee3347006/bertly.py#L74) – we do this to make sure that `https://www.dosomething.org` and `https://www.dosomething.org/` would resolve to the same shortened link (since they're the same URL, just formatted differently).

I'll make a ticket to address this before we launch, but shouldn't block the backfill (since all of the existing Bertly 1.0 shortened URLs have already been normalized by that [`url_normalize`](https://pypi.org/project/url-normalize/) Python package).